upstream-community-operators/ovms-operator/0.1.0/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM scratch
+
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=ovms-operator
+LABEL operators.operatorframework.io.bundle.channels.v1=alpha
```

Comment:
@dtrawins Since you have own docker file you will have to add label also here. 
```
LABEL operators.operatorframework.io.bundle.channel.default.v1=alpha
```

Reply:
Thank you @mvalarh. I added the missing labels for operators.operatorframework.io.bundle.channel.default.v1 and com.redhat.openshift.versions